hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -655,11 +656,16 @@
       CompactionState compactionState = admin.getCompactionState(table.getName()).get();
       %><%= compactionState %><%
     } catch (Exception e) {
-      // Nothing really to do here
-      for(StackTraceElement element : e.getStackTrace()) {
-        %><%= StringEscapeUtils.escapeHtml4(element.toString()) %><%
+
+      if(e.getCause().getCause() instanceof NotServingRegionException) {
```

Comment:
`if(e.getCause() != null && e.getCause().getCause() instanceof NotServingRegionException)`

Reply:
yes @virajjasani . I updated the PR.